runtimevar/runtimevar.go

```diff
@@ -91,16 +90,33 @@ func (c *Variable) Watch(ctx context.Context) (Snapshot, error) {
 		c.prev = cur
 		v, err := cur.Value()
 		if err != nil {
-			// Mask underlying errors.
-			return Snapshot{}, fmt.Errorf("Variable.Watch: %v", err)
+			return Snapshot{}, wrapError(err)
```

Comment:
The UI doesn't let me comment on L77, but should L77 also wraps the error?

Reply:
No, that isn't necessary, because that's guaranteed to be a context-canceled error, not an error from a provider.